prague-postgres-meetup-oct-2024.mdx

```diff
@@ -0,0 +1,116 @@
+The meetup ends with a closing talk and some networking.
+
+<Alert flexDirection="column" alignItems="flex-start" gap="1">
+  <b>18:00 | Opening talk by Tomas Vondra</b>
+</Alert>
+
+<Alert flexDirection="column" alignItems="flex-start" gap="1">
+  <b>18:15 | ORM and Lambdas use with Postgres by Ondrej Hlavac</b>
+</Alert>
+
+<Alert flexDirection="column" alignItems="flex-start" gap="1">
+  <b>19:00 | PostgreSQL and DuckDB: Analytics Performance and Use Cases Compared by Josef Machytka</b>
+</Alert>
+
+<Alert flexDirection="column" alignItems="flex-start" gap="1">
+  <b>20:00 | Closing, end of event</b>
+</Alert>
+
+### Session highlights
+
+<Alert flexDirection="column" alignItems="flex-start" gap="1">
+  <b>ORM and Lambdas use with Postgres | Ondrej Hlavac</b>
+  <Box as="span" color="titleAccent">
+    In this talk, Ondrej will explore how they integrate AWS Lambda functions with PostgreSQL on RDS, leveraging ORM
+    techniques in their Python API services. He will cover how they manage Lambda connections to the database using
+    connection pooling with AWS RDS Proxy and how they store logs and metrics in Postgres for system monitoring and
+    performance optimization.
+  </Box>
+</Alert>
+
+<Alert flexDirection="column" alignItems="flex-start" gap="1">
+  <b>PostgreSQL and DuckDB: Analytics Performance and Use Cases Compared | Josef Machytka</b>
+  <Box as="span" color="titleAccent">
+    The talk will explore the capabilities and features of PostgreSQL and DuckDB in the context of data analytics. It
+    will highlight key differences, focusing on the strengths and weaknesses of each database for analytical workloads.
+    The presentation will also include benchmark results from some typical business use cases, providing real-world
+    performance comparisons. Additionally, we'll discuss how to combine PostgreSQL and DuckDB effectively in data
+    analytics pipelines to leverage the best of both worlds.
+  </Box>
+</Alert>
+
+## Prague PostgreSQL Developer Day 2025
+
+We also announced [P2D2 (Prague PostgreSQL Developer Day)](https://p2d2.cz/) 2025 last week in Athens!  
+The 17th P2D2 will be held on January 28-29, 2025, at ČVUT FIT, Thákurova 9, Prague.
+
+The conference spans two days:
+
+- January 28. 2025 (Tuesday) - workshops/trainings, i.e. half-day or whole-day sessions for smaller groups of attendees
+- January 29, 2025 (Wednesday) - regular conference with 45-minute talks
+
+📣 The calls for [talks and trainings](https://docs.google.com/forms/d/1jm13YksXYUiM3dlwGjHPWMQwVGro4tQxrFYVoOP9PK8/viewform?edit_requested=true) and [sponsors](https://docs.google.com/forms/d/1Uq-lsYP3nj4w9KQGFyguOtZDSr8HYT26rbiMl47PjdY/viewform?edit_requested=true) are now open.
+Talk and training proposals are accepted until November 23, 2024 (Sunday, end of day). In case of questions, let us know at info@p2d2.cz
+
+## Conclusion
+
+Hope to see everyone in and around Prague joining us on the 29th of October for the meetup!
```

Comment:
If there are plans to record the meetup presentations, might be worth mentioning here.

Reply:
We don't record, though wish we did. Sometimes the venues have the recording options and we use it.